server/model/monitor.js

```diff
@@ -13,12 +13,24 @@ const {Notification} = require("../notification")
 const monitor_response_time = new Prometheus.Gauge({
     name: 'monitor_response_time',
     help: 'Monitor Response Time (ms)',
-    labelNames: ['monitor_name']
+    labelNames: [
+        'monitor_name',
+        'monitor_type',
+        'monitor_url',
+        'monitor_hostname',
+        'monitor_port'
+    ]
 });
 const monitor_status = new Prometheus.Gauge({
     name: 'montor_status',
     help: 'Monitor Status (1 = UP, 0= DOWN)',
-    labelNames: ['monitor_name']
+    labelNames: [
+        'monitor_name',
+        'monitor_type',
+        'monitor_url',
+        'monitor_hostname',
+        'monitor_port'
```

Comment:
Duplicated labels - line 17-21

Reply:
@Saibamen - not sure how else to do this, it's a different monitor object compared to the one on lines 17-21?

I'm all in favour of DRY, but I'm not a JS dev so I've no idea what the JS preference is around this kind of thing?

Appreciate the feedback though :)

Reply:
```
const commonLabels = [
    'monitor_name',
    'monitor_type',
    'monitor_url',
    'monitor_hostname',
    'monitor_port',
]
```

And in `monitor_response_time` and `monitor_status` use:
`labelNames: commonLabels,`

FYI: @louislam

Reply:
And if anyone wanted to add new label, but only for `monitor_response_time`, he can use spread syntax:
```javascript
labelNames: [
    ...commonLabels,
    'new_label',
]
```

Reply:
This is now done.